stories/Input.stories.tsx

```diff
@@ -1,11 +1,75 @@
 /* eslint-disable import/no-extraneous-dependencies */
-import React, { useState, SyntheticEvent } from 'react';
+import React, { useState, SyntheticEvent, FunctionComponent } from 'react';
 import { storiesOf } from '@storybook/react';
 import centered from '@storybook/addon-centered';
 import { Input, Form, Button } from '../src';
 
 const stories = storiesOf('Input', module);
 
+interface Props {
+  disableErrorFromComponents?: boolean;
+}
+
+const ExampleForm: FunctionComponent<Props> = (props: Props) => {
```

Comment:
The prop typing is clever when specifying a function as a functional component, so you don't need to declare `(props: Props)` as well as the FunctionComponent, it's very minor though!

Reply:
Good point, have removed it now